### PR TITLE
[HDX-8544] enable dev containers for vscode

### PR DESCRIPTION
**Pull request by teodorescuserban**

### CODE REVIEW

File: ckan/cli/cli.py [hunk 1]
```diff
@@ -221,3 +221,6 @@ def ckan():
 ckan.add_command(less.less)
 ckan.add_command(generate.generate)
 ckan.add_command(user.user)
+
+if __name__ == "__main__":
```
Comment (teodorescuserban) [inline ckan/cli/cli.py]: ping @alexandru-m-g - see https://github.com/ckan/ckan/discussions/6457#discussioncomment-1436466

Comment (alexandru-m-g) [reply ckan/cli/cli.py]: @teodorescuserban Can you explain a bit why this is needed ? We were running _ckan run_ before without needing this. Also, this change is not merged in CKAN core.

**Comment by alexandru-m-g** [comment]
There's already a schema.xml file in the repo.
https://github.com/OCHA-DAP/hdx-ckan/blob/dev/ckanext-hdx_search/ckanext/hdx_search/hdx-solr/schema.xml
We should use this one.

File: ckan/cli/cli.py (hunk 1 above)
Comment (teodorescuserban) [reply ckan/cli/cli.py]: I'll manage without it.

**Comment by teodorescuserban** [comment]
> There's already a schema.xml file in the repo. https://github.com/OCHA-DAP/hdx-ckan/blob/dev/ckanext-hdx_search/ckanext/hdx_search/hdx-solr/schema.xml We should use this one.

@alexandru-m-g, can we also add the other solr files required in a configset in the same location?

**Pull request merged**